bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/NetatmoBindingConstants.java

```diff
@@ -95,6 +95,8 @@
     public static final ThingTypeUID WELCOME_HOME_THING_TYPE = new ThingTypeUID(BINDING_ID, "NAWelcomeHome");
     public static final ThingTypeUID WELCOME_CAMERA_THING_TYPE = new ThingTypeUID(BINDING_ID, "NACamera");
     public static final ThingTypeUID WELCOME_PERSON_THING_TYPE = new ThingTypeUID(BINDING_ID, "NAWelcomePerson");
+    // Presence camera
+    public static final ThingTypeUID PRESENCE_CAMERA_THING_TYPE = new ThingTypeUID(BINDING_ID, "NOC");
```

Comment:
"NOC" is a type id which comes from the Netatmo API, I can't change that. ;-) At least the constant name is more meaningful. See https://dev.netatmo.com/apidocumentation/security#products-details

Reply:
We have the full control / choice of our openHAB thing types.
As I understand, until now the OH thing type is matching the API type ?
There was no real reason for that and as soon as this type has no easy meaning for the end user, I would suggest to use a more meaning thing type, like OutdoorCamera for example.
And it was a bad idea to use uppercase letters in the thing types. But it is too late to change that.

Reply:
"NOC" has technically to match the Netatmo API type, there is no other mapping mechanizm within the binding.

Yes, Netatmo changed their product names. "Welcome" is now "Indoor", and "Presence" is now "Outdoor". I decide to keep the old names, because everything is named "Welcome". I think it would be a good idea to plan a separate refactoring issue to rename everything to the new names, "Welcome" is existing in nearly every class, so that will become expensive and should planned separately.

Reply:
> "NOC" has technically to match the Netatmo API type

Can you please point in the code where it is required to have the openHAB thing type being named "NOC" to be able to use the API ?

Reply:
Method "org.openhab.binding.netatmo.internal.discovery.NetatmoModuleDiscoveryService#discoverWelcomeHome", line 149. When you follow the trace you will find method "org.openhab.binding.netatmo.internal.discovery.NetatmoModuleDiscoveryService#findThingUID", there it is matched with the constants. See also the usage of method "io.swagger.client.model.NAWelcomeCamera#getType", there comes "NOC" from the Netatmo API like described in the API documentation from Netatmo.

Reply:
Ok, you're absolutely right ! I don't like how it was done but it makes sense in this case to continue with the same patten and so use NOC as thing type in openHAB.